src/modules/transaction/utils/encoding.js

```diff
@@ -13,8 +13,7 @@
  *
  */
 
-import { codec } from '@liskhq/lisk-codec';
-import { utils } from '@liskhq/lisk-cryptography';
+import { codec as liskCode, cryptography  } from '@liskhq/lisk-client';
```

Comment:
This should be `liskCodec` not `liskCode` I guess @oskarleonard 
```suggestion
import { codec as liskCodec, cryptography  } from '@liskhq/lisk-client';
```

Reply:
Fixed